### ISSUE TITLE
Mobs not staying in boats

### ISSUE THREAD

**Issue by honeydewhq** [PrideCraft/Public-Issue-Tracker#17]
Describe the bug
When I get into a boat that has 1 mob in it the mob jumps out of the boat.

To Reproduce
1. Get a boat
2. Get 1 mob into the boat
3. Right click to get in the boat
4. See the mob is no longer in the boat

Expected behavior
The mob to stay in the boat when I also sit in it

Screenshots
N/A

Username
Please include your Minecraft username, Discord username, Tag, and ID ([don't know how to get your id?](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-))
honeydewhq, honeydewhq#3239, 675124212553220111

**Comment by Notarin** [comment]
Hi @honeydewhq could you provide screenshots and timestamps?

**Comment by CoolPanda101** [comment]
Correct. For some reason one person or entity is allowed in a boat, maybe to prevent duping? I'm not sure its not something I have change since the ownership change

**Comment by Notarin** [comment]
@honeydewhq we still need some screenshots and timestamps
